apps/explorer/src/app/routes/network-parameters/network-parameters.test.tsx

```diff
@@ -0,0 +1,80 @@
+import { render, screen } from '@testing-library/react';
+import { NetworkParametersTable } from './network-parameters';
+import type { NetworkParametersQuery } from './__generated__/NetworkParametersQuery';
+
+describe('NetworkParametersTable', () => {
+  it('renders correctly when it has network params', () => {
+    const data: NetworkParametersQuery = {
+      networkParameters: [
+        {
+          __typename: 'NetworkParameter',
+          key: 'market.liquidityProvision.minLpStakeQuantumMultiple',
+          value: '1',
+        },
+        {
+          __typename: 'NetworkParameter',
+          key: 'market.fee.factors.infrastructureFee',
+          value: '0.0005',
+        },
+      ],
+    };
+    const { container } = render(
```

Comment:
chore (blocking): prefer screen to container

https://kentcdodds.com/blog/common-mistakes-with-react-testing-library#using-container-to-query-for-elements

Reply:
resolved in last commit